src/web/app/pages-admin/admin-home-page/admin-home-page.module.ts

```diff
@@ -27,6 +28,7 @@ const routes: Routes = [
     FormsModule,
     RouterModule.forChild(routes),
     AjaxLoadingModule,
+    NewInstructorDataRowModule,
```

Comment:
If this is the only place where the module is used, probably a new module isn't needed. The new component can be part of the admin home page module.

Reply:
Ok, I will move it.

Reply:
This module is still there. You only moved the files to the same folder (or rather a subfolder within it), but you're still using a different module which is not necessary.

Reply:
Oh wait, right. Sorry, I think I got confused. Hang on, I will remove the module.